### PR TITLE
Schema, README, extension metadata

### PR DESCRIPTION
**Pull request by ColinMaudry**
I wasn't able to complete the checks locally as the URL in extension.json `documentationUrl` returned an error. It may be due to current Github's  network issues.

### CODE REVIEW

**Comment by ColinMaudry** [comment]
I've tried with https://extensions.open-contracting.org/en/extensions/tenderClassification, same result.

**Comment by jpmckinney** [comment]
Aha, it's because you accidentally created a private repo. I made it public now, and I've disabled the creation of new private repos :)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
-# ocds_tenderClassification_extension
+# Tender classification 
+
 Adds an array of Classification objects to Tender in order to categorize the contracting process.
+
+## Guidance
```
Comment (jpmckinney) [inline README.md:5]: Empty section.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
-# ocds_tenderClassification_extension
+# Tender classification 
+
 Adds an array of Classification objects to Tender in order to categorize the contracting process.
```
Comment (jpmckinney) [inline README.md:3]: > Adds an array of classification objects to the tender object, in order to categorize the procedure as a whole.

We can also describe the relationship to `tender.items`:

> The items to be procured are expected to have more specific classifications than the procedure as a whole.

**Review by jpmckinney — COMMENTED** [review]
I'll make the quick changes. Just noting as FYI.

**Pull request merged**